public/app/plugins/panel/graph/graph.ts

```diff
@@ -334,8 +334,8 @@ class GraphElement {
         let bucketSize: number;
 
         if (this.data.length) {
-          const histMin = _.min(_.map(this.data, s => s.stats.min));
-          const histMax = _.max(_.map(this.data, s => s.stats.max));
+          const histMin = panel.xaxis.min ? panel.xaxis.min : _.min(_.map(this.data, s => s.stats.min));
```

Comment:
if panel.xaxis.min is 0 then this check is not working :)    (A common mistake we have done many times in Grafana)

Reply:
As a matter of fact, that is true. I prepare a fix.